packages/dx-react-grid-material-ui/src/templates/toolbar.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Toolbar as ToolbarMUI } from 'material-ui';
+import { withStyles } from 'material-ui/styles';
+import { darken, fade, lighten } from 'material-ui/styles/colorManipulator';
+
+const styles = theme => ({
+  toolbar: {
+    paddingTop: theme.spacing.unit * 1.5,
+    borderBottom: `1px solid ${
```

Comment:
Is this border needed according to the material guidelines?

Reply:
There are [two](https://storage.googleapis.com/material-design/publish/material_v_12/assets/0B3mOPoJlxiFhemNvbnZOcXNpODQ/components-datatables-cards-altheaders.png) variants. As for me, the first one looks better.

Reply:
Ok. Let's leave them as is

Reply:
But the second one doesn't require extra styles))